auth build without pandoc is broken after #4790

```
Making all in docs
make[2]: Entering directory `/home/kees/src/pdns/docs'
echo "You need pandoc to generate the manpages"
echo "You need pandoc to generate the manpages"
echo "You need pandoc to generate the manpages"
echo "You need pandoc to generate the manpages"
echo "You need pandoc to generate the manpages"
echo "You need pandoc to generate the manpages"
You need pandoc to generate the manpages
exit 1
You need pandoc to generate the manpages
exit 1
You need pandoc to generate the manpages
You need pandoc to generate the manpages
exit 1
You need pandoc to generate the manpages
exit 1
exit 1
You need pandoc to generate the manpages
exit 1
make[2]: *** [pdns_server.1] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [pdns_control.1] Error 1
make[2]: *** [zone2ldap.1] Error 1
make[2]: *** [pdnsutil.1] Error 1
make[2]: *** [zone2json.1] Error 1
make[2]: *** [zone2sql.1] Error 1
make[2]: Leaving directory `/home/kees/src/pdns/docs'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/kees/src/pdns'
make: *** [all] Error 2
```

--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -53,8 +53,7 @@ clean:
 else
 if !HAVE_MANPAGES
 $(MANPAGES_TARGET_ALL):
-	echo "You need pandoc to generate the manpages"
-	exit 1
+	@echo "Pandoc is required to build $@"
 endif
 endif
 

